server/controllers/Followers.js

```diff
@@ -161,6 +176,25 @@ class Followers {
     }
     return false;
   }
+
+  /**
+   * @name UpdateUserFollowersAndFollowingsCount
+   * @async
+   * @static
+   * @memberof Followers
+   * @param {Object} source the source user object
+   * @param {Object} target the target user object
+   * @returns {Null} Null object
+   */
+  static async UpdateUserFollowersAndFollowingsCount(source, target) {
```

Comment:
and this

Reply:
@dinobi I will change the method name accordingly.